docs/DeveloperGuide.adoc

```diff
@@ -279,51 +287,88 @@ Refer to the guide <<DevOps#, here>>.
 
 *Target user profile*:
 
+* Human Resource Admin In-Charge of School of Computing 'Hackathon' Event
 * has a need to manage a significant number of contacts
+* has a need to register participants in bulk
+* has a need to classify contacts into Mentor, Participants and Teams
+* has a need to keep track of which member is in which Team
+* has a need to keep track of the seating positions of each Team
+* has a need to keep track of Mentor assignments to Teams
+* has a need to keep track of the competition winners and prizes won
+* has a need to search for specific Mentor, Team or Participant at times
 * prefer desktop apps over other types
 * can type fast
 * prefers typing over mouse input
 * is reasonably comfortable using CLI apps
 
-*Value proposition*: manage contacts faster than a typical mouse/GUI driven app
+*Value proposition*:
+
+* manage contacts faster than a typical mouse/GUI driven app
+* keep track of the relationship between Participant, Team and Mentor, such that it can be referenced at times
+* store a significant number of contacts in an organised, readable manner
 
 [appendix]
 == User Stories
 
 Priorities: High (must have) - `* * \*`, Medium (nice to have) - `* \*`, Low (unlikely to have) - `*`
 
-[width="59%",cols="22%,<23%,<25%,<30%",options="header",]
+[width="79%",cols="15%,<20%,<35%,<37%",options="header",]
 |=======================================================================
 |Priority |As a ... |I want to ... |So that I can...
 |`* * *` |new user |see usage instructions |refer to instructions when I forget how to use the App
 
-|`* * *` |user |add a new person |
+|`* * *` |Admin In-Charge |find a Entity by name |locate details of Entity without having to go through the entire list
+
+|`* * *` |Admin In-Charge |delete an Entity by name |remove entries I no longer need
+
+|`* * *` |Admin In-Charge |add an Entity by name and contact information |update the list of Entities
+
+|`* * *` |Admin In-Charge |updated an Entity by name and contact information |update the specific entries
+
+|`* * *` |Admin In-Charge |register individuals en-masse(with provided registration information) | avoid tedious manual registration
+
+|`* * *` |Admin In-Charge |keep track of winning teams and the prizes won | ensure that the prize-giving ceremony runs smoothly
+
+|`* * *` |Admin In-Charge |keep track of winning teams and the prizes won | ensure that the prize-giving ceremony runs smoothly
+
+|`* * *` |Admin In-Charge |make sure that I will be notified on any wrong commands that I type | make sure that I do not accidentally clutter up my list of entries
```

Comment:
If I understood correctly, the notification for wrong commands should be "so that I can re-enter the correct command".